### ISSUE TITLE
suggestions

### ISSUE THREAD

**Issue by NikitaKhromov-Borisov** [jasp-stats/jasp-desktop#1443]
1. I fell it would be fruitful to report p-values as exact (real) as possible, not as inequality <.001. It's extremely important!!!
2. Contrasts should be presented with confidence (better credible) intervals.
3. For contingency tables analysis it would be desirable to have an opportunity to receive BF01, BF10 and log(BF10) simultaneously as well BFs for all four (five) sampling models at once like in the case of Post Hoc Tests in ANOVA.
4. It would desirable to have and confidence (and/or credible ?) intervals for Cohen's effect size as well as for contingency coefficients.
5. When presenting the results of correlation analysis JASP draws the scatter plot with a straight line. If it's a regression line then it can be misleading because when both correlated variables are random then there must be two regression lines: for regression X by Y and vice-versa Y by X. 
6. So it seems to be reasonable to draw confidence (credible) ellipses (instead of straight lines) like those produced by software PAST: http://folk.uio.no/ohammer/past/
7. I try to use your fresh beta JASP version. However, when with the attached data I check 
   Descriptive plots the following report appears:
   ![image](https://cloud.githubusercontent.com/assets/16406847/18177222/e5571628-7081-11e6-9dac-86d6f5005155.png)
8. Once again, I hope that you will accept my suggestions to report real p-values, not inequalities p<0.001.
   This is a great disadvantage of JASP.
   Nikita Khromov-Borisov

**Comment by FransMeerhoff** [comment]
Hello @NikitaKhromov-Borisov 

We appreciate your suggestions a lot.
We've assigned your issues and we will let you know the result.

About the bug:
We tried this with the attached data on the latest version and could not reproduce your problem so your problem should be fixed in the official JASP 0.8.0.0. release.
We plan to release the official JASP 0.8.0.0 in a few weeks.

Kind regards
Frans Meerhoff

[Pt_IPC-Diox_HIPEC.txt](https://github.com/jasp-stats/jasp-desktop/files/456298/Pt_IPC-Diox_HIPEC.txt)
